### PR TITLE
scripts(ticdc): add a parameter to continue integration test

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Add a new parameter to continue intergration tests from the failed one, 
for  example:  if we run `make  integration_test_mysql` and failed at `common_1`,  then we can continue it with command
`make integration_test_mysql START_AT="common_1" `

Issue Number: ref #3881

### Check List 

Tests 

 - Integration test
 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by hi-rustin — COMMENTED** [review]
Maybe `START_AT` is better?

**Comment by hi-rustin** [comment]
rest LGTM

File: tests/integration_tests/README.md [hunk 1]
```diff
@@ -76,7 +78,7 @@ We recommend that you provide docker with at least 6+ cores and 8G+ memory. Of c
 > We will try to resolve these issues as soon as possible.
 
 1. If you want to run kafka tests,
-   run `CASE="clustered_index" docker-compose -f ./deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml up --build`
+   run `START_AT="clustered_index" docker-compose -f ./deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml up --build`
```
Comment (asddongmen) [inline tests/integration_tests/README.md:81]: Why only change `run kafka test` to `START_AT`?

Comment (sdojjy) [reply tests/integration_tests/README.md:81]: to show the usage of `CASE` and `START_AT`t

File: tests/integration_tests/run.sh [hunk 2]
```diff
@@ -52,18 +52,33 @@ test_case=$2
...
+
+start_at=$3
+if [ -z "$start_at" ]; then
+	start_at="*"
+fi
+
 set -eu
 
-if [ "$test_case" == "*" ]; then
+continue_test="no"
```
Comment (asddongmen) [inline tests/integration_tests/run.sh:63]: May be `run_test` is a better name.

Comment (sdojjy) [reply tests/integration_tests/run.sh]: done

**Review by asddongmen — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**